packages/plugin-vite/src/plugins/patches/commonjs_test.ts

```diff
@@ -719,3 +706,62 @@ var __importDefault = this && this.__importDefault || function (mod) {
 const node_events_1 = __importDefault(_mod.default ?? _mod);`,
   });
 });
+
+Deno.test("commonjs imitating esm - default export exists", () => {
+  runTest({
+    input: `var replace = String.prototype.replace;
```

Comment:
The license of this code is BSD-3-clause, shouldn't cause any problem.

https://github.com/ljharb/qs/blob/main/lib/formats.js

Reply:
Let's simplify the fixture to the relevant bits only. Makes our tests less noisy too.